blog/_posts/2017-05-23-whats-new-scala-native.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: blog
+post-type: blog
+by: Martin Duhem, Guillaume Massé and Denys Shabalin
+title: "What's new with Scala Native?"
+---
+
+It has been a little bit more than two months since Scala Native 0.1 has been
+released. What's new in Scala Native? What should you expect for future
+releases?
+
+## Scala Native 0.2
+
+In April, [Scala Native 0.2 was released][scala-native-0.2-release]. The main
+focus of this release was to increase the coverage of classes from
+the JDK, such that that more programs can be ported to Scala Native without any further
+effort.
```

Comment:
Since you go on to detail the stuff you added to Scala Native, I'd add a sentence at the end here which better leads into the rest of the article. Because it doesn't really connect well at the moment.

E.g.,:

What parts of Java do we now support in Scala Native? Lots! We've added support for IO and regular expressions: